src/scripts/settings-theme.js

```diff
@@ -0,0 +1,2 @@
+const darkMode = JSON.parse(localStorage.getItem('darkMode'));
```

Comment:
```suggestion
const darkMode = JSON.parse(localStorage.getItem('darkMode')) ?? window.matchMedia('(prefers-color-scheme: dark)').matches;
```
Should we default to system-theme?

Reply:
should flow through from here right? https://github.com/toggl/toggl-button/pull/1769/files#diff-e7905509b504970b523439849d059794R12

Reply:
Not for first load I think, but that's fine best not to duplicate code and make the logic more obscure.